GameDefinition/Content/ModeKind.hs

```diff
@@ -231,10 +231,10 @@ ambush = ModeKind  -- dense ranged with reaction fire vs melee at night
   , mcaves  = cavesAmbush
   , mendMsg = []
   , mrules  = T.unlines $
-      [ "* one level only"
-      , "* 3 heroes that shoot concurrently, unspecified human foes"
-      , "* avoid losses"
-      , "* assert control of the situation ASAP"
+      [ "* One level only"
+      , "* Three heroes capable of concurrent action vs. unspecified number of human enemies"
```

Comment:
okay, so maybe 
> Three heroes capable of concurrent shooting/attacking vs. Various unknown enemies

From the comment above, I'm not sure if they are capable of both shooting and melee. In that case 'action' ~> 'attack', otherwise 'action' ~> 'shooting'

Reply:
Yes, both melee and shoot.

How about expressing "unspecified number and identity", but in a less verbose way?

Reply:
How does it fit on the screen?

Reply:
It doesn't fit well. I just took a look and it breaks the line, so that is unacceptable 
![image](https://user-images.githubusercontent.com/68469446/87970127-e4a46a00-caf5-11ea-8cb8-4b065fb31ade.png)
I could do the following and it would fit:
"enemies" ~> "foes"
"capable of" ~> "with
![image](https://user-images.githubusercontent.com/68469446/87970695-afe4e280-caf6-11ea-949a-1dd63a32e236.png)

Reply:
How about the shortened version, but with 'vs. Unidentified foes"?

Reply:
so "Three heroes with concurrent attacks vs. Unidentified foes"? Yes that would be nice and simple actually. I would just update the previous lines to use foes

Reply:
Yes. Regarding "enemies", I think they are fine and this lets us differentiate the known enemies with the mysterious foes even more and so set the mood for the scenario.

As soon as you are done, please feel free to merge. Don't wait for CI, it takes too long.